torch/_inductor/constant_folding.py

```diff
@@ -374,7 +374,7 @@ def untag(node: torch.fx.Node) -> bool:
     # We rewrite the tags, if it's a constant being directly consumed, without
     # any folding opportunity, we keep it in main gm.
     for node in gm.graph.nodes:
-        if node.op == "getattr" or (node.name in (lifted_constant_names or ())):
+        if node.op == "get_attr" or (node.name in (lifted_constant_names or ())):
```

Comment:
What do you think of pulling these magic strings out as constants like META_TAG is?